datadog_checks_base/datadog_checks/base/checks/win/wmi/types.py

```diff
@@ -0,0 +1,11 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from collections import namedtuple
+from typing import Any, Dict, List, Tuple, Union
+
+WMIMetric = namedtuple('WMIMetric', ['name', 'value', 'tags'])
```

Comment:
If relevant, note that there's a typing-compatible syntax for namedtuples:

```suggestion
from typing import NamedTuple

WMIMetric = NamedTuple('WMIMetric', [('name', str), ('value', float), ('tags', List[str])])
```